docs/explanations/dialects.md

```diff
@@ -7,16 +7,16 @@ In contrast to traditional infrastructure for compilers, the set of operations a
 
 Concrete compiler takes advantage of these concepts by defining a set of dialects, capable of representing an FHE program from an abstract specification that is independent of the actual cryptosystem down to a program that can easily be mapped to function calls of a cryptographic library. The dialects for the representation of an FHE program are:
 
-* The FHELinalg Dialect ([documentation](FHELinalgDialect.md), [source](https://github.com/zama-ai/concrete/tree/main/compilers/concrete-compiler/compiler/include/concretelang/Dialect/FHELinalg/IR/FHELinalgOps.td))
```

Comment:
bad when we 
- branch a CP version for gitbook
- and then later, change things in `main`
so I changed it (and it's now tested by the new script)